Restructure GCP and Kube-resilience faults (#680)
harness/developer-hub: docs/chaos-engineering/chaos-faults/gcp/gcp-vm-instance-stop-by-label.md

      </tr>
      <tr>
        <td> GCP_PROJECT_ID </td>

>         <td> ID of the GCP project, to which the VM instances belong. </td>

Reviewer: ```suggestion
        <td>Id of the GCP project containing the VM instances. </td>
```
